### ISSUE TITLE
[Preview]: Enable Azure Monitor for VMs with Azure Monitoring Agent(AMA) issue for Azure compute gallery images

### ISSUE THREAD

**Issue by MarcelHeek** [Azure/Enterprise-Scale#1300]
The (preview) policy has a parameter : `Optional: List of VM images that have supported Windows OS to add to scope`

The info icon shows the following information: 
![image](https://user-images.githubusercontent.com/84451910/235430128-94a1228e-f795-4104-a59c-e57a6e077041.png)

It apparently offers the ability to have this policy initiative triggered for VM images. I would expect that the initiative would also get triggered for VM images that are stored in a Azure Computer Gallery, however it does not.

**Steps to reproduce**

1. Assign the AMA policy initiative to e.g. a subscription scope. 
2. Deploy an image from an Azure Compute Gallery, based on 2022-datacenter sku
3. The policy initiative will DINE the user assigned identity to the VM
4. The policy initiative will deploy the Data Collection Rule
5. The policy initiative does not detect the deployed VM image as an incompliant resource and no AMA installation is triggered.

**Screenshots**
The VM image definition;
![image](https://user-images.githubusercontent.com/84451910/235431444-50f71529-7582-4d40-83b6-79728454a14b.png)

The assignment parameters;
![image](https://user-images.githubusercontent.com/84451910/235431613-05a10a12-da89-4359-b1b5-928baae5a433.png)

One VM deployed from Azure Compute Gallery, on 2022-datacenter sku deployed from market place . One gets the AMA agent deployed. The other is not detected as a target resource.
![image](https://user-images.githubusercontent.com/84451910/235432496-4598037c-011d-43ad-a24b-ef811a89a5c0.png)

**Comment by Springstone** [comment]
Hi @MarcelHeek. Please note we have little influence on built-in policies and cannot fix these. I would recommend you open a support ticket in Azure for this issue.

That being said, this is a known issue and impacts custom gallery images as well as migrated virtual machines. There is an update to this policy on the way that will allow you to skip the OS/SKU check (a flag is provided), which should arrive very soon.

**Comment by Springstone** [comment]
I see a number of the AMA policies have already been updated, look for the property: scopeToSupportedImages like in this policy https://www.azadvertizer.net/azpolicyinitiativesadvertizer/9575b8b7-78ab-4281-b53b-d3c1ace2260b.html

**Comment by MarcelHeek** [comment]
Hi @Springstone Thanks for responding. We are consuming the CAF-ES terraform module(s) to put everything in as code, I got a reference to the initiative I raised the issue for in the issues logged for the lack of support for AMA.

The policy `[Preview]: Enable Azure Monitor for VMs with Azure Monitoring Agent(AMA)` looks like the best starting point for integration into the policy archetype(s) so I hope this can be fixed very soon.
